init/src/components/{{componentPath}}.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-export default class <%= componentName %> extends React.Component {
+export class <%= componentName %> extends React.Component {
```

Comment:
This could be a stateless functional component.

Reply:
I'm going to leave this one as it's just a skeleton example for the template.

Reply:
👍